### PR TITLE
feat(delete-tag): get default remote from git config if exists

### LINKED ISSUE #870: cmd `delete-tag` assumes "origin" as remote for tags
`delete-tag` the pushes the against the hard-coded remote 'origin' .

This failed for me b/c i renamed this usually to 'o' for laziness' sake ;)

`delete branch` figures the remote out from the branche's remote which doens't work for tags.
However, the current branch's remote could be used as a fall back which to me would make sense in most cases.

Another idea were to make the remote a CLI option or could be even put into the git config.
Note sure if this issue comes up in other extra commands as well in which case a consistent behavior across all commands would be appreciated

### PR DESCRIPTION
**Pull request by vanpipy**
fix #870

### CODE REVIEW

File: bin/git-delete-tag [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
 test -z $1 && echo "tag required." 1>&2 && exit 1
 
+# Set the default origin
+origin="origin"
+
+# Detect the defalt remote exist or not
+defualt_remote=$(git config git-extras.default-remote)
+
+if [[ ! -z "$defualt_remote" ]]; then
+    origin="$defualt_remote"
```
Comment (SuperSandro2000) [inline bin/git-delete-tag:13]:
```suggestion
default_remote=$(git config git-extras.default-remote)

if [[ -n "$default_remote" ]]; then
    origin="$default_remote"
else
   origin=origin
fi
```

File: bin/git-delete-tag [hunk 2]
```diff
@@ -3,6 +3,16 @@
 # Assert there is at least one tag provided
 test -z $1 && echo "tag required." 1>&2 && exit 1
 
+# Set the default origin
+origin="origin"
+
+# Detect the defalt remote exist or not
```
Comment (SuperSandro2000) [inline bin/git-delete-tag:9]:
```suggestion
# Detect the defalt remote exists or not
```

File: bin/git-delete-tag [hunk 3]
```diff
@@ -3,6 +3,16 @@
 # Assert there is at least one tag provided
 test -z $1 && echo "tag required." 1>&2 && exit 1
 
+# Set the default origin
+origin="origin"
+
```
Comment (SuperSandro2000) [inline bin/git-delete-tag:8]:
```suggestion
```

File: bin/git-delete-tag (hunk 1 above)
Comment (vanpipy) [reply bin/git-delete-tag]: Update.

File: bin/git-delete-tag (hunk 2 above)
Comment (vanpipy) [reply bin/git-delete-tag]: Correct the typo and update

File: bin/git-delete-tag (hunk 3 above)
Comment (vanpipy) [reply bin/git-delete-tag]: Move the `origin` into the `else` and update.

**Pull request merged**